python/lsst/summit/extras/slewTimingSimonyi.py

```diff
@@ -0,0 +1,351 @@
+    # 'lsst.sal.MTHexapod.logevent_configuration',
+    "lsst.sal.MTHexapod.logevent_configurationApplied",
+    # 'lsst.sal.MTHexapod.logevent_configurationsAvailable',
+    # 'lsst.sal.MTHexapod.logevent_connected',
+    # 'lsst.sal.MTHexapod.logevent_controllerState',
+    "lsst.sal.MTHexapod.logevent_errorCode",
+    # 'lsst.sal.MTHexapod.logevent_heartbeat',
+    "lsst.sal.MTHexapod.logevent_inPosition",
+    # 'lsst.sal.MTHexapod.logevent_interlock',
+    # 'lsst.sal.MTHexapod.logevent_logLevel',
+    # 'lsst.sal.MTHexapod.logevent_logMessage',
+    # 'lsst.sal.MTHexapod.logevent_simulationMode',
+    # 'lsst.sal.MTHexapod.logevent_softwareVersions',
+    # 'lsst.sal.MTHexapod.logevent_summaryState',
+    # 'lsst.sal.MTHexapod.logevent_uncompensatedPosition'
+]
+
+inPositionTopics = {
+    "Hexapod": "lsst.sal.MTHexapod.logevent_inPosition",
+    "M2": "lsst.sal.MTM2.logevent_m2AssemblyInPosition",
+    "Azimuth": "lsst.sal.MTMount.logevent_azimuthInPosition",
+    "Camera cable wrap": "lsst.sal.MTMount.logevent_cameraCableWrapInPosition",
+    "Elevation": "lsst.sal.MTMount.logevent_elevationInPosition",
+    "Rotator": "lsst.sal.MTRotator.logevent_inPosition",
+}
+
+
+def plotExposureTiming(
+    client: EfdClient,
+    expRecords: list[dafButler.DimensionRecord],
+    plotHexapod: bool = False,
+    prePadding: float = 1,
+    postPadding: float = 3,
+) -> matplotlib.figure.Figure:
+    """Plot the mount command timings for a set of exposures.
+
+    This function plots the mount position data for the entire time range of
+    the exposures, regardless of whether the exposures are contiguous or not.
+    The exposures are shaded in the plot to indicate the time range for each
+    integration its readout, and any commands issued during the time range are
+    plotted as vertical lines.
+
+    Parameters
+    ----------
+    client : `EfdClient`
+        The client object used to retrieve EFD data.
+    expRecords : `list` of `lsst.daf.butler.DimensionRecord`
+        A list of exposure records to plot. The timings will be plotted from
+        the start of the first exposure to the end of the last exposure,
+        regardless of whether intermediate exposures are included.
+    plotHexapod : `bool`, optional
+        Plot the ATAOS.logevent_hexapodCorrectionStarted and
+        ATAOS.logevent_hexapodCorrectionCompleted transitions?
+    prePadding : `float`, optional
+        The amount of time to pad before the start of the first exposure.
+    postPadding : `float`, optional
+        The amount of time to pad after the end of the last exposure.
+
+    Returns
+    -------
+    fig : `matplotlib.figure.Figure`
+        The figure containing the plot.
+    """
+    inPositionAlpha = 0.5
+    commandAlpha = 0.5
+    integrationColor = "grey"
+    readoutColor = "blue"
+
+    legendHandles = []
+
+    expRecords = sorted(expRecords, key=lambda x: (x.day_obs, x.seq_num))  # ensure we're sorted
```

Comment:
Since you've already got a list and don't want a copy, just use `expRecords.sort(...)` to do it in place.